src/Microsoft.DotNet.VersionTools/Dependencies/MSBuildPackageReferenceUpdater.cs

```diff
@@ -0,0 +1,191 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.DotNet.VersionTools.Util;
+using NuGet.Packaging.Core;
+using NuGet.Versioning;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Xml;
+using System.Xml.Linq;
+
+namespace Microsoft.DotNet.VersionTools.Dependencies
+{
+    public class MSBuildPackageReferenceUpdater : IDependencyUpdater
+    {
+        public IEnumerable<string> MSBuildFilePaths { get; }
+
+        public MSBuildPackageReferenceUpdater(IEnumerable<string> msbuildFilePaths)
```

Comment:
Hm, it's a good point, and I think we discussed this yesterday in person. We would still need some "updater task" that ran on the global properties file, and we'd need to come up with some sort of naming convention, right? I have seen other repos with a pattern like:

```XML
<PackageReference Include="MyPackageName" Version="$(MyPackageNameVersion") />
<PackageReference Include="OtherPackageName" Version="$(OtherPackageNameVersion") />
```